index.html

```diff
@@ -0,0 +1,444 @@
+                    <img class="team-card__image" src="./assets/img/team2.png" alt="">
+                    <div class="team-card__name">
+                        Rodney Stratton
+                    </div>
+                    <p class="team-card__title">
+                        Consultant
+                    </p>
+                    <div class="social">
+                        <img class="social__link" src="./assets/svgs/fbvector.svg" alt="facebook">
+                        <img class="social__link" src="./assets/svgs/instavesctor.svg" alt="instagram">
+                        <img class="social__link" src="./assets/svgs/twitterVector.svg" alt="twitter">
+                    </div>
+                </div>
+                <div class="team-card">
+                    <img class="team-card__image" src="./assets/img/team3.png" alt="">
+                    <div class="team-card__name">
+                        Avie Beaton
+                    </div>
+                    <p class="team-card__title">
+                        CO Founder
+                    </p>
+                    <div class="social social--spacing-md">
+                        <img class="social__link" src="./assets/svgs/fbvector.svg" alt="facebook">
+                        <img class="social__link" src="./assets/svgs/instavesctor.svg" alt="instagram">
+                        <img class="social__link" src="./assets/svgs/twitterVector.svg" alt="twitter">
+                    </div>
+                </div>
+            </div>
+        </div>
+    
+        <div class="contact-us contact-us--spacing-midium ">
+            <div class="container container--spacing-small">
+                <div class="contact-us__title">
+                    Contact Us
+                </div>
+                <h1 class="site-header site-header--spacing-sm" >
+                    Make an Appointment
+                </h1>
+            </div>
+            <div class="container container--spacing-small">
+                <form accept="#" class="contact-us__form">
+                    <input class="contact-us__field" type="text" placeholder="Fullname">
+                    <input class="contact-us__field" type="email" placeholder="example@email.com">
+                    <select class="contact-us__field" id="">
+                        <option value="">Please selct</option>
+                    </select>
+                    <select class="contact-us__field" id="">
+                        <option value="">04:00 Available</option>
+                    </select>
+                    <textarea class="contact-us__field" name="" id="" cols="30" rows="10" placeholder="Message"></textarea>
+                    <button class="button button--primary button--rounded button--size-small" >
+                        Book Appointment
+                    </button>
+                </form>
+            </div>
+        </div>
+
+        <div class="footer footer--dark">
+            <div class="container">
```

Comment:
`container` element is not referenced no where , you can improve the naming here